types/block.go

```diff
@@ -1381,7 +1381,7 @@ func DataFromProto(dp *cmtproto.Data) (Data, error) {
 
 // -----------------------------------------------------------------------------
 
-// EvidenceData contains any evidence of malicious wrong-doing by validators.
+// EvidenceData contains any evidence of misbehavior committed by a validator.
```

Comment:
This is a list of evidence. See the documentation for `EvidenceList`. Everything else is for memoization (caching).

Reply:
thanks, re-worded